services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None,
```

Comment:
If the default value is None - you should use Optional[str] or str | None typehint.

Reply:
fix it in all code